### ISSUE TITLE
Port view to d3

### ISSUE THREAD

**Issue by ongardie-sfdc** [salesforce/runway-model-elevators#1]
Using React in Runway is deprecated (SalesforceEng/runway-browser#1)

**Comment by joelburget** [comment]
I agree with the desire to remove big dependencies, but what I don't understand is why it's necessary to bundle d3, react, or any other rendering library. I'm much, much more likely to use runway if I can use react. Anyway, I don't think it makes sense to be prescriptive, rather you should encourage users to bundle whatever libraries they need.

I'm guessing the reason for pre-bundling dependencies is to enable dynamic loading of code from github using browser-require, etc. That's a valid point in the design space, but it restricts library use, and my version of safari fails to load https://runway.systems, because `let` was used. Bundling with webpack / browserify _is_ an extra step, but avoids both problems.

**Comment by ongardie-sfdc** [comment]
@joelburget, those are good thoughts. I think this applies to Runway in general, and I don't want this getting lost with the elevator model. Let me reply over on salesforce/runway-browser#1.
